src/classes/PPlan_Primary_TDTM.cls

```diff
@@ -51,27 +58,50 @@ public with sharing class PPlan_Primary_TDTM extends TDTM_Runnable {
         Set<Id> primaryProgramIds = new Set<Id>();
         // BEFORE INSERT
         if ( newlist != null && triggerAction == TDTM_Runnable.Action.BeforeInsert && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_Before_Insert)) {
+            for (integer i=0; i<newlist.size(); i++) {
+                Program_Plan__c newPPlan = (Program_Plan__c)newlist[i];
+                if (newPPlan.Is_Primary__c == true) {
+                    // Make Plan "Current" if "Is Primary" is checked.
+					newPPlan.Status__c = STATUS_CURRENT_VAL;
+                }
+            }
+        }
+        // AFTER INSERT
+        if ( newlist != null && triggerAction == TDTM_Runnable.Action.AfterInsert && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_After_Insert)) {
             for (integer i=0; i<newlist.size(); i++) {
                 Program_Plan__c newPPlan = (Program_Plan__c)newlist[i];
                 if (newPPlan.Is_Primary__c == true) {
                     //if multiple program plans under one single account are set as primary in same transation
                     //Use the first program plan as the primary
-                    if (!acccountIdsNeedNonPrimarycount.contains(newPPlan.Account__c)) {
+					if (!acccountIdsNeedNonPrimarycount.contains(newPPlan.Account__c)) {
```

Comment:
Looks like this line is the same except for spacing. I'm thinking the IDE is sometimes using tabs and other time using spaces. I think we try to ensure that tabs are converted to spaces via the IDE. In any case, just verify spacing.